component-state.md

```diff
@@ -281,6 +281,10 @@
   https://github.com/dabapps/react-set-props  
   Store arbitrary state in redux with a similar API to setState
   
+- **encaps**  
+  https://github.com/megazazik/encaps
```

Comment:
This looks like it needs two spaces after the link:

```
  https://github.com/megazazik/encaps<space><space>
```

Reply:
I added spaces